hive/src/main/java/com/netease/arctic/hive/catalog/ArcticHiveCatalog.java

```diff
@@ -362,7 +362,7 @@ private org.apache.hadoop.hive.metastore.api.Table newHiveTable(TableMeta meta)
       org.apache.hadoop.hive.metastore.api.Table newTable = new org.apache.hadoop.hive.metastore.api.Table(
           meta.getTableIdentifier().getTableName(),
           meta.getTableIdentifier().getDatabase(),
-          System.getProperty("user.name"),
+          meta.getProperties().getOrDefault("owner", System.getProperty("user.name")),
```

Comment:
Use table properties constants. 
set this properties when create table at spark code.